waku/v2/protocol/waku_store/waku_store_types.nim

```diff
@@ -26,6 +26,8 @@ type
     ## This type is used to encapsulate a WakuMessage and its Index
     msg*: WakuMessage
     index*: Index
+    # TODO may need to rename this object as it holds both the index and the pubcub topic of a waku message
```

Comment:
```suggestion
    # TODO may need to rename this object as it holds both the index and the pubsub topic of a waku message
```

Not sure I follow this

Reply:
The `IndexedWakuMessage` was initially used to hold a waku message with its index and the type was named `IndexedWakuMessage` due to this. However, now that I added the `pubsubTopic` field to it, I thought that the type name `IndexedWakuMessage` is no longer a good representative of what it embodies. This TODO is to keep a reminder of renaming it in the future (specially when more data fields get included in this struct).

Reply:
I see, I'd probably add that comment closer to object. Now that I skim it I read it as being related to the pubsubTopic field below

Reply:
Sure done